hyperon_das_atomdb/adapters/ram_only.py

```diff
@@ -327,30 +311,20 @@ def get_matched_type(
                 return self._filter_non_toplevel(templates_matched)
         return templates_matched
 
-    def get_atom_as_dict(
-        self, handle: str, arity: Optional[int] = 0
-    ) -> Dict[str, Any]:
-        atom = self.db.node.get(handle)
-        if atom is not None:
-            return {
-                'handle': atom['_id'],
-                'type': atom['named_type'],
-                'name': atom['name'],
-            }
-        atom = self._get_link(handle)
-        if atom is not None:
-            return {
-                'handle': atom['_id'],
-                'type': atom['named_type'],
-                'template': self._build_named_type_template(
-                    atom['composite_type']
-                ),
-                'targets': self.get_link_targets(atom['_id']),
-            }
-        raise AtomDoesNotExistException(
-            message='This atom does not exist',
-            details=f'handle: {handle}',
-        )
+    def get_atom(self, handle: str) -> Dict[str, Any]:
+        atom_type = 'node'
+        document = self.db.node.get(handle)
+        if document is None:
+            atom_type = 'link'
+            document = self._get_link(handle)
+        atom = self._replace_keys(document, atom_type)
```

Comment:
I see no reason to keep links with `key_0`, `key_1`, ... `key_n` instead of `targets`. This is relevant in `redis_mongo` because MongoDB can make better indexing in separate fields but it's pointless here. Storing `targets` instead of separate keys would avoid the construction of the new dict in `_replace_keys()` and this is relevant because memory allocation is an expensive operation for RAM data structures and this method particularly will be called A LOT.

You don't need to change this for this PR if you don't want to but please add a `medium` priority issue to make this later.